jlm/llvm/opt/alias-analyses/PointerObjectSet.cpp

```diff
@@ -1006,9 +1093,9 @@ PointerObjectConstraintSet::CreateOvsSubsetGraph()
   std::vector<util::HashSet<PointerObjectIndex>> successors(totalNodeCount);
   std::vector<bool> isDirectNode(totalNodeCount, false);
 
-  // Initialize all registers as direct nodes
+  // Initialize all registers with empty points-to sets as direct nodes
   for (auto [_, index] : Set_.GetRegisterMap())
-    isDirectNode[index] = true;
+    isDirectNode[index] = Set_.GetPointsToSet(index).IsEmpty();
```

Comment:
The `isEmpty()` suggests to me that a bool is returned, but the comment above tells me that it is an empty points-to set. If the `IsEmpty()` method does not return a bool, then maybe just call it `Empty()` or `EmptySet()`?

Reply:
This is a bool, but the comment is worded poorly. Fixing